src/screens/signup/index.tsx

```diff
@@ -13,11 +14,13 @@ interface SignupScreenProp extends NavigationInterface {
 
 export default function SignupScreen(props: SignupScreenProp) {
   const { colors } = useThemeContext();
+  const [email, setEmail] = useState("")
   return (
     <Container>
       <SafeAreaView>
       <Welcome>Welcome</Welcome>
       <Prompt>SIGN UP OR LOG IN</Prompt>
```

Comment:
use small letter here and set text-transform to uppercase its easy that way to manipulate texts on the app other than changing the text itself to be uppercase